packages/dataviews/src/types.ts

```diff
@@ -0,0 +1,84 @@
+/**
+ * External dependencies
+ */
+import type { ReactNode } from 'react';
+
+type Item = Record< string, any >;
+
+interface Option {
+	value: any;
+	label: string;
+}
+
+interface filterByConfig {
+	operators?: string[];
+	isPrimary?: boolean;
+}
+
+export interface Field {
+	/**
+	 * The unique identifier of the field.
+	 */
+	id: string;
+
+	/**
+	 * The label of the field. Defaults to the id.
+	 */
+	header?: string;
```

Comment:
I think we should rename this one "label". header is too "table" specific wording.